api/v1beta1/helmchart_types.go

```diff
@@ -171,7 +171,6 @@ func (in *HelmChart) GetInterval() metav1.Duration {
 // +genclient
 // +genclient:Namespaced
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:shortName=hc
```

Comment:
Keeping this one is fine I think, as it is not a variant of `*repository` (nor conflicts with any of our other resource kinds).

Reply:
> Can you please rebase instead of merging changes in, and smash your commits (as it is a small change)?

Done 👍